.github/workflows/worker-deploy.yml

```diff
@@ -8,6 +8,8 @@ jobs:
   deploy:
     runs-on: ubuntu-latest
     name: Deploy
+    environment: ${{ github.ref == 'refs/heads/main' && 'main' || 'development' }}
```

Comment:
Hard coding the branch name is suspicious again.

Reply:
Will there be any other deployments other than the development and main branch?

Reply:
We support any branch name to be targeted from the kernel config